findspam.py

```diff
@@ -59,6 +59,13 @@
     "microsoft", "newegg", "cnet", "regex101", r"(?<!plus\.)google", "localhost", "ubuntu", "getbootstrap",
     r"jsfiddle\.net", r"codepen\.io", "pastebin"
 ] + [se_dom.replace(".", r"\.") for se_dom in SE_SITES_DOMAINS])))
+ASN_WHITELISTED_WEBSITES = ["docs.unity3d.com", "trac.ffmpeg.org", "bitcoincore.org", "ffmpeg.org",
+    "latex.codecogs.com", "www.advancedcustomfields.com", "name.com", "businessbloomer.com",
```

Comment:
Flake8 is suggesting that these four lines should be indented to be level with the start of the square brackets

Reply:
Thanks for weighing in on that one- I wasn't sure about that as the whitelist above had a different indent level :)